src/main/java/net/dv8tion/jda/api/managers/RoleManager.java

```diff
@@ -273,6 +282,32 @@ default RoleManager setColor(@Nullable Color color)
     @CheckReturnValue
     RoleManager setMentionable(boolean mentionable);
 
+    /**
+     * Sets the {@link net.dv8tion.jda.api.entities.Icon Icon} of this {@link net.dv8tion.jda.api.entities.Role Role}.
+     *
+     * @param  icon
+     *         The new icon for this {@link net.dv8tion.jda.api.entities.Role Role}
+     *         or {@code null} to reset
+     *
+     * @return RoleManager for chaining convenience
+     */
+    @Nonnull
+    @CheckReturnValue
+    RoleManager setIcon(@Nullable Icon icon);
+
+    /**
+     * Sets the Unicode Emoji of this {@link net.dv8tion.jda.api.entities.Role Role}.
```

Comment:
The docs don't really make it clear to me what the difference between emoji and icon is

Reply:
I'd like to know that too without paying discord to see the feature - it's all I got from the docs: https://github.com/discord/discord-api-docs/blob/master/docs/resources/Guild.md#create-guild-role--post-guildsguildiddocs_resources_guildguild-objectroles